src/components/theme/Header/ToggleTheme/index.jsx

```diff
@@ -1,15 +1,16 @@
-import React, { useContext } from 'react';
-import { ThemeContext } from 'providers/ThemeProvider';
-import sunIcon from 'assets/icons/sun.svg';
-import moonIcon from 'assets/icons/moon.svg';
-import { Wrapper } from './styles';
+import React, { useContext } from "react";
+import { ThemeContext } from "providers/ThemeProvider";
+import sunIcon from "assets/icons/sun.svg";
+import sunIcon-inverted from "assets/icons/sun-inverted.svg";
```

Comment:
Hi mate, you have a syntax error here

```suggestion
import sunIconInverted from "assets/icons/sun-inverted.svg";
```

(Although I think this file isn't needed. You can instead just apply `mix-blend-mode: difference` to the current .svg file & it'll invert itself depending on whatever value you give it)

Reply:
Yeah. I removed this file and added a ```filter: invert()``` to the ```<img>``` element to solve the issue. I didn't use ```mix-blend-mode: difference``` as it was already being used with the hover effect.